apps/web/src/components/FiatOnRampModal/FiatOnRampModal.tsx

```diff
@@ -216,7 +234,7 @@ export const FiatOnRampModal = memo<InjectedModalProps & FiatOnRampProps>(functi
     } finally {
       setTimeout(() => setLoading(false), 2000)
```

Comment:
Why we need to wait for like 2 seconds to clear the loading?

Reply:
Can we use `react-query` when dealing with asynchronous request? So you don't need to handle the loading state yourself